ansible/tasks/setup-ci-environ.yml

```diff
@@ -1,32 +1,34 @@
 ---
-  - name: Ensure {{ __test_harness_environ }} configmap is present
-    include_tasks: create-configmap.yml
+- name: Ensure {{ __test_harness_environ }} configmap is present
+  include_tasks: create-configmap.yml
 
-  - name: Block for tasks that only run when not using test_harness_dry_run
-    when: not test_harness_dry_run
-    block:
-      # NOTE: This is not idempotent - it will report changed every time
-      # seems to be a problem in the k8s module
-      - name: Ensure {{ __test_harness_environ }} objects are present
-        k8s:
-          src: "{{ __test_harness_ci_obj_file }}"
-          namespace: "{{ test_harness_namespace }}"
-          state: present
+- name: create the openshift objects file from the template
+  template:
+    src: "{{ __test_harness_ci_obj_file | d(test_harness_openshift_template) }}"
```

Comment:
There's no place to define __test_harness_ci_obj_file in this pr? Is it needed?

Reply:
removed `__test_harness_ci_obj_file` in https://github.com/linux-system-roles/test-harness/pull/95/commits/5238051d32d4de4232b63fe441a6ab2681a747f7